manifests/kube-events-exporter-cluster-role.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     app.kubernetes.io/component: events-exporter
     app.kubernetes.io/name: kube-events-exporter
-    app.kubernetes.io/version: 0.1.0
+    app.kubernetes.io/version: v0.1.0
```

Comment:
Version shouldn't have `v` prefix

Reply:
Indeed, thank you for pointing that out.